ui/app/AppLayouts/Chat/views/CommunityColumnView.qml

```diff
@@ -94,12 +94,25 @@ Item {
         }
     }
 
+    StatusButton {
+        id: joinCommunityButton
+        text: root.communityData.access === Constants.communityChatOnRequestAccess
```

Comment:
This looks like it's checking against request access for a specific channel, instead of the entire community.
Can you verify you're checking against the correct value?

Reply:
@PascalPrecht I used the same as in `InvitationBubble` and so on:

Reply:
And `communityData` its `activeSection`